src/main/java/frc/robot/common/LEDDriver.java

```diff
@@ -0,0 +1,13 @@
+package frc.robot.common;
+
+import edu.wpi.first.wpilibj.Spark;
+
+public class LEDDriver extends Spark {
+
+    public LEDDriver(int port) {
+        super(port);
+    }
+
+    public static final double autonomous = -0.99;
+    public static final double teleop = -0.61;
```

Comment:
Since these are the definitions for your constants, you should capitalize these since you use the capitalized version elsewhere and since they're constants.